.teamcity/_self/projects/WPComTests.kt

```diff
@@ -40,6 +40,9 @@ object WPComTests : Project({
 	// Gutenberg Atomic
 	buildType(gutenbergPlaywrightBuildType("desktop", "c341e9b9-1118-48e9-a569-325100f5fd9" , atomic=true, edge=false));
 	buildType(gutenbergPlaywrightBuildType("mobile", "e0f7e412-ae6c-41d3-9eec-c57c94dd8385", atomic=true, edge=false));
+	// Gutenberg Atomic Edge
+	buildType(gutenbergPlaywrightBuildType("desktop", "4c66d90d-99c6-4ecb-9507-18bc2f44b551" , atomic=true, edge=true));
```

Comment:
Oh, does Kotlin support passing arguments with this named parameter syntax without further changes in the method signature? I don't remember passing these as named parameters before!

Reply:
I changed it in the previous PR https://github.com/Automattic/wp-calypso/pull/63035 to named parameters, to make it clearer instead of a bunch of unnamed params.

Reply:
Nice!